experimental/Schemes/src/ToricBlowups/constructors.jl

```diff
@@ -61,45 +64,52 @@ end
     blow_up(v::NormalToricVariety, new_ray::AbstractVector{<:IntegerUnion}; coordinate_name::String = "e")
 
 Blow up the toric variety by subdividing the fan of the variety with the
-provided new ray. Note that this ray must be a primitive element in the
-lattice Z^d, with d the dimension of the fan. This function returns the
-corresponding blowdown morphism.
+provided new ray. This function returns the corresponding morphism.
+
+Note that this ray must be a primitive element in the lattice Z^d that
+does not coincide with any existing ray, with d the dimension of the
+fan. In particular, it is currently impossible to blow up along a ray
+which corresponds to a non-Q-Cartier divisor.
 
 By default, we pick "e" as the name of the homogeneous coordinate for
-the exceptional divisor. As third optional argument one can supply
-a custom variable name.
+the exceptional prime divisor. As optional argument one can supply a
+custom variable name.
+
+!!! warning
+    This function is type unstable. The type of the field `center_unnormalized` is always a subtype of `AbsIdealSheaf` (meaning that `center_unnormalized(f) isa Oscar.AbsIdealSheaf` is always true). Sometimes, the function computes and sets the field `center_unnormalized` for the output `f`, giving it the type `ToricIdealSheafFromCoxRingIdeal` (meaning that `center_unnormalized(f) isa Oscar.ToricIdealSheafFromCoxRingIdeal` is true and `center_unnormalized(f) isa IdealSheaf` is false). If it does not, then calling `center_unnormalized(f)` computes and sets the field `center_unnormalized` and it will have the type `IdealSheaf` (meaning that `center_unnormalized(f) isa Oscar.ToricIdealSheafFromCoxRingIdeal` is false and `center_unnormalized(f) isa IdealSheaf` is true).
 
 # Examples
+
+In the example below `f.center_unnormalized` has type `ToricIdealSheafFromCoxRingIdeal` and we can access the corresponding ideal in the Cox ring using `Oscar.ideal_in_cox_ring`.
```

Comment:
```suggestion
In the example below `center_unnormalized(f)` has type `ToricIdealSheafFromCoxRingIdeal` and we can access the corresponding ideal in the Cox ring using `Oscar.ideal_in_cox_ring`.
```
wouldn't it make more sense to guide users to the getter function instead of the direct access?